src/main/java/Main.java

```diff
@@ -1,8 +1,12 @@
 import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.exception.DaoAnnotationNotFound;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
**do we need this import? (KseniiaMakarova asked)**
my answer:
_I'm not sure, that I clearly understand you.
import controller.ConsoleHandler -> ConsoleHandler handler = ;
import core.basesyntax.lib.Injector -> Injector.
import java.lang.reflect.InvocationTargetException -> exception in signature main_